aws/resource_aws_workspaces_directory.go

```diff
@@ -62,6 +62,36 @@ func resourceAwsWorkspacesDirectory() *schema.Resource {
 					},
 				},
 			},
+			"workspace_creation_properties": {
```

Comment:
can you add `enable_work_docs` as well?

Reply:
Unfortunately, there is no API for updating this attribute. You can set it only once during directory registration.

Reply:
ForceNew?

Reply:
Probably yes... 🤔  let me try this...

Reply:
Fun fact: You can enable WorkDocs on WorkSpaces directory registration, but you can break this connection later only from WorkDocs's side (deleting the site). Or with entire WorkSpace directory deregistration of course. I'd prefer to have uni-directional management dependency between services.

Reply:
I was wrong, it's possible to modify WorkDocs enable status with [`ModifyWorkspaceCreationProperties`](https://docs.aws.amazon.com/workspaces/latest/api/API_WorkspaceCreationProperties.html#WorkSpaces-Type-WorkspaceCreationProperties-EnableWorkDocs) API action now. I have to note, `enable_work_docs` creates new WorkDocs site in case of absence, but doesn't delete it after turning off. Anyway, I drop `ForceNew` for this attribute.

Reply:
Cool, i think we can just document this point.

Reply:
Unfortunately, there is no WorkDocs API for sites. That means we can't clean up and sweep implicitly created WorkDocs site in tests. I've requested WorkDocs site CRUD API from AWS Support.

Propose to roll out `workspace_creation_properties` without `enable_work_docs` attribute, leaving it enabled by default, because you can change this setting only by [`ModifyWorkspaceCreationProperties`](https://docs.aws.amazon.com/workspaces/latest/api/API_WorkspaceCreationProperties.html#WorkSpaces-Type-WorkspaceCreationProperties-EnableWorkDocs) API call. This behavior will be covered in documentation until AWS releases the required API and we develop `aws_work_docs_site` resource.

Reply:
Done